Package.swift

```diff
@@ -10,35 +10,15 @@ let package = Package(
         .iOS(.v14)
     ],
     products: [
-        .library(name: "Runestone", targets: ["Runestone"])
+        .library(name: "Runestone", targets: ["Runestone"]),
+    ],
+    dependencies: [
+        .package(url: "https://github.com/tree-sitter/tree-sitter", branch: "master")
```

Comment:
Would it make sense to pin to a specific version of Tree-sitter rather than the main branch? We should be able to pin to version 0.20.8, the latest version of Tree-sitter, with this version of Runestone.

Reply:
Just realized that we can't pin against a specific version until a new version of Tree-sitter that includes the Package.swift has been published. Guess we'll have to pin against the master branch, or alternatively a specific commit, until then.